deploy/examples/cert-manager/README.md

```diff
@@ -16,11 +16,11 @@
 
 2. Provision certificates:
 
-        kubectl apply -f selfsigned-ca.yaml -n istio-system
+        kubectl apply -f deploy/examples/cert-manager/selfsigned-ca.yaml -n istio-system
 
 3. Install cert-manager istio-csr Service:
 
-        helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr -f deploy-prototype/istio-csr-helm-values.yaml"
+        helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr -f deploy-prototype/istio-csr-helm-values.yaml
```

Comment:
This path doesn't look correct.

I think we need to decide from where users should be invoking these commands.  If they are from the same directory as this readme, they should all be `./`.  If they are from the project root, then `./deploy/examples/cert-manager/` would be correct.

Reply:
deploy/examples/cert-manager/

It looks good, we can user anywhere with the correct path.